pkg/kritis/review/review.go

```diff
@@ -129,8 +204,8 @@ func (r Reviewer) ReviewISP(images []string, isps []v1beta1.ImageSecurityPolicy,
 			if len(violations) != 0 {
 				return r.handleViolations(image, pod, violations)
 			}
-			if r.config.IsWebhook {
-				if err := r.addAttestations(image, isp, notAttestedBy, c); err != nil {
+			if r.config.IsWebhook && auth != nil {
```

Comment:
ditto here, and maybe have a well-named variable hold `auth != nil` value?

Reply:
Maybe then let's use `authProvided := auth != nil`? And then use it here and above?

Reply:
good idea. done.